build.gradle

```diff
@@ -29,6 +29,21 @@ allprojects {
 
 apply plugin: 'java'
 
+/* SET PROJECT INFO for all artifacts */
+assert hasProperty('version')   // See version in gradle.properties.
+System.out.println("version=${version}")
```

Comment:
First let me say Thanks for the Contribution!  I'm new to gradle so excuse nubie questions.

I don't understand the value of this assert & println.
All projects have a 'version' property so the assert will never fail.
The println prints `version=unspecified` because the property hasn't been set -- gradle.properties defines `build_version` (later in the script it sets a subproject's version to build_version  -- I don't know why the author chose to do it that way).
Should the above lines just be removed / were they for your initial debugging?

Regarding 'group', it feels to me like it should be handled to version / build_version, originating from gradle.properties.  Your thought on that?